uvobot.rb

```diff
@@ -1,51 +1,24 @@
-require 'curb'
-require 'nokogiri'
-require 'json'
-require 'uri'
 require 'dotenv'
+require './slack_notifier'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.